lib/safe-pg-migrations/base.rb

```diff
@@ -70,8 +71,16 @@ def config
   end
 
   module Migration
+    module ClassMethods
+      def safe_pg_migrations_verbose(verbose = nil)
```

Comment:
Should
```ruby
safe_pg_migrations_verbose
```
really be a valid call?
I'd rather not have a default value here.

Reply:
It's an equivalent of your proposition in the second comment: without a parameter, `safe_pg_migrations_verbose` without an attribute is the reader, `safe_pg_migrations_verbose true` is the writer.

Sadly I cannot use both your attr_reader proposition and the API you propose, because attr_reader defines the method `def safe_pg_migrations_verbose` without a parameter.

Maybe the correct solution is to adapt the API?